packages/cli-lib/errorHandlers.js

```diff
@@ -26,4 +31,6 @@ module.exports = {
   logApiUploadErrorInstance,
   logFileSystemErrorInstance,
   logServerlessFunctionApiErrorInstance,
+  FieldErrors,
```

Comment:
Nit, can we add something like `module` or `cms` somewhere in the naming of these two utils? The use cases of the CLI are expanding so I'd like to create as much clarity as possible through naming.